<launch>
    <group ns="rover6" >
        <arg name="motors_topic" default="motors_throttle"/>
        <arg name="servos_topic" default="servos_throttle"/>
        <!-- <arg name="motors_topic" default="motors"/> -->
        <!-- <arg name="servos_topic" default="servos"/> -->

        <node type="throttle" name="motor_topic_throttle" pkg="topic_tools" required="true" output="screen"
            args="messages motors 15.0 $(arg motors_topic)"/>
        <node type="throttle" name="servo_topic_throttle" pkg="topic_tools" required="true" output="screen"
            args="messages servos 15.0 $(arg servos_topic)"/>

        <param name="serial_port" type="string" value="/dev/ttyTHS1"/>
        <param name="serial_baud" type="int" value="115200"/>
        <param name="motors_topic" type="string" value="$(arg motors_topic)"/>
        <param name="servos_topic" type="string" value="$(arg servos_topic)"/>
        <param name="imu_frame_id" type="string" value="imu"/>
        <node type="rover6_serial_bridge_node" name="rover6_serial_bridge_node" pkg="rover6_serial_bridge" required="true" output="screen"/>
    </group>
</launch>
